assignment1.txt

```diff
@@ -0,0 +1,9 @@
+Christopher Liu
+1. The return 0; at the end of the main() signifies that the function has completed with error 0, which is no error. Technically it is not needed for most compilers.
+2. the *r at the end should be /r as otherwise it will multiply instead of divide twice
```

Comment:
I think this is your answer for number 3, not 2, but you're right nevertheless - you can also fix this with parentheses. You are missing a couple of more errors - see if you can find them!